libs/openFrameworks/types/ofColor.cpp

```diff
@@ -169,7 +169,7 @@ A clampedMultiply(const A& a, const B& b) {
 }
 
 template<typename PixelType>
-float ofColor_<PixelType>::limit() {
+PixelType ofColor_<PixelType>::limit() {
```

Comment:
this is kind of dangerous, i'm pretty sure in some places we are doing things like:
```
value/limit()
```

to normalize a value and this change will break those places for integer numbers, casting this to float would avoid that and the warning.

in general i'm a little nervous about this changes of the types being used, if it's to solve warnings i guess we need to do it but otherwise i would hold until 0.9 is released so we have more time to test. it's usually really big commits and even if most changes are innocuous, things like this can really easily slip through.